### ISSUE TITLE
govlab logo

### ISSUE THREAD

**Issue by dino-at-thegovlab** [govlabacademy/govlabacademy.github.io#119]
Shouldn't the govlab logo go at the end together with the knight one?

**Comment by claudioccm** [comment]
nope. Knight foundation is a sponsor of the GovLab. The GovLab Academy is a
product of the GovLab.
